web/conf.d/app.conf

```diff
@@ -24,6 +24,11 @@ server {
         dav_methods PUT;
     }
 
+    location /statevector/gateway/state_election.txt {
+        alias /usr/share/nginx/statevector/gateway/state_election.txt;
+        dav_methods PUT;
```

Comment:
toto prosim ta znamena, ze iba PUT na to cestu vies robit? Lebo robit tam aj GET request, na zistenie stavu

Reply:
Nie, pridáva to PUT, ale funguje naďalej aj GET.